### PR TITLE
upbound-cli: Add version 0.11.0

### PR DESCRIPTION
**Pull request by stevehipwell**
This PR add the Upbound CLI `up` to Scoop.

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/upbound-cli.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "architecture": {
+        "64bit": {
+            "url": "https://cli.upbound.io/stable/v0.11.0/bin/windows_amd64/up.exe",
+            "hash": "47e08b86a3f3c40652fa2d51dd8e27d85c7e113fdfeb4f02fc20896072baab9b"
+        }
+    },
+    "bin": "up.exe",
+    "checkver": {
+        "github": "https://github.com/upbound/up"
+    },
```
Comment (issaclin32) [inline bucket/upbound-cli.json:15]:
```suggestion
    "checkver": {
        "url": "https://api.github.com/repos/upbound/up/tags",
        "regex": "tags/v([\\d.]+)\""
    },
```

**Comment by stevehipwell** [comment]
> This package does not meet the Main criteria yet

@rashil2000 so the bot says but it works locally using `checkver.ps1` which is how I generate the url and hash when adding a new package.

**Comment by rashil2000** [comment]
> > This package does not meet the Main criteria yet
> 
> @rashil2000 so the bot says but it works locally using `checkver.ps1` which is how I generate the url and hash when adding a new package.

The bot does not check the eligibility criteria (yet 😅). It has to be checked manually. See the template: https://github.com/ScoopInstaller/Main/blob/master/.github/ISSUE_TEMPLATE/Package_request.md

**Comment by stevehipwell** [comment]
@rashil2000 I saw the discussion but I didn't think you were serious about using GH stars and forks as a gate!? Stars tend to be biased towards hot new dev tools and don't really say anything about actual popularity/use. Forks are probably even more meaningless; it's good practice to delete a fork after your PR has been merged, and even ignoring that there are plenty of projects out there in the "stadium" category which don't accept PRs so will have hardly any forks.

**Comment by stevehipwell** [comment]
@rashil2000 I didn't respond because I honestly thought it was an idle suggestion which would obviously be seen as an incorrect basis to judge package use/quality! What is your timeframe for removing packages already in main which don't meet this criteria? I also assume that you are monitoring last updated and static analysis for all packages as both of these are far better indicators of quality than stars or forks!? What about the individual supply chain security?

I thought common sense would prevail, so without better metrics each package should be a subjective decision. I guess I'll have to start another maintainer discussion as this feels all to familiar to how the project was operating when I pestered @rasa so much he started adding new maintainers.

In this case 60M+ of funding for a tool built on the hugely popular Crossplane project says that this would be a relevant addition to main.

**Comment by rashil2000** [comment]
> @rashil2000 I didn't respond because I honestly thought it was an idle suggestion which would obviously be seen as an incorrect basis to judge package use/quality! 

An idle suggestion that was agreed upon by 3 other maintainers? If you want to voice something, participate in the discussion.

> What is your timeframe for removing packages already in main which don't meet this criteria?

None. Existing packages will not be removed.

> I thought common sense would prevail, so without better metrics each package should be a subjective decision. I guess I'll have to start another maintainer discussion as this feels all to familiar to how the project was operating when I pestered @rasa so much he started adding new maintainers.

Is this a warning/threat? I'm a firm believer in [AGF](https://en.wikipedia.org/wiki/Wikipedia:Assume_good_faith) but please convince me that the language used here (and [elsewhere](https://github.com/ScoopInstaller/Main/pull/3513#issuecomment-1103811359)) does not indicate a warning.

> In this case 60M+ of funding for a tool built on the hugely popular Crossplane project says that this would be a relevant addition to main.

This information would have been sufficient (and should be mentioned in the PR description). There really was no need for pointing fingers or being pessimistic about the state of the project.

**Comment by stevehipwell** [comment]
> An idle suggestion that was agreed upon by 3 other maintainers? If you want to voice something, participate in the discussion.

I don't think 3 makes a quorum, I'd have responded if I thought there was a majority in favour. I'm pretty sure GH stars have been debunked as an indicator of project quality many many times.

> None. Existing packages will not be removed.

In which case it's disingenuous to suggest the new policy is based on security. Current supply chain thinking would have 2FA, signing and SBOM as the first steps for this.

> Is this a warning/threat? I'm a firm believer in AGF but please convince me that the language used here (and elsewhere) does not indicate a warning.

I'm not sure how you could take that as a warning or threat? It's an observation that arbitrary decisions are being made by the minority without any objective data to back them up; much like how the project was previously run. As a maintainer I'm suggesting that we have further discussions about what the process should be for major policy decisions as I don't think this one was in the best interest of the project.

> This information would have been sufficient (and should be mentioned in the PR description). There really was no need for pointing fingers or being pessimistic about the state of the project. Scoop is doing fantastic and is seeing more activity than it has ever seen.

If this information is required then the PR template should ask for it. IMHO that's a better solution than arbitrary stars etc. Lightweight flexible processes would be far more suitable for a contributor led project like this.

**Comment by rashil2000** [comment]
> If this information is required then the PR template should ask for it. 


It does ask for it. Any PR requires either a corresponding bug report or a package request before being raised. And the package request template asks for any and all signs of popularity (not just stars). 

People raise neither (bug report or package request), but I don't really sweat it - that is unless the signs of well-known-ability aren't immediately obvious

**Pull request merged**